packages/page-accounts/src/Sidebar/Sidebar.tsx

```diff
@@ -121,11 +130,26 @@ export default React.memo(styled(FullSidebar)`
 
   .ui--AddressMenu-addr {
     word-break: break-all;
-    width: 26ch;
+    width: 24ch;
```

Comment:
Ok, thanks @jacogr.
Should I consult Gav about this change or we are ok with it?  
this was needed to have the "Copied" text well aligned after clicking on "Copy".
Also, Displaying the whole address in the Sidebar without an explicit action from the user looks to me unsecured.
Any youtuber recording a tutorial or blogger taking a screenshot will have his address revealed to the world.
So I can "hide" the whole address by shortening the address display to:
<fisrst five characters>...<last five characters>

Like other wallets do (Metamask, Keplr, Binance wallet,...)

What do you think of it?

Reply:
format : to "first five letters"..."last five letters" of the address which is 13 characters all in all

Reply:
Could it be that some browsers restrict copying smth automatically?
I personally feel safer when I see the whole address instead of `"first five letters"..."last five letters"`, otherwise I'm starting to ask why smth is hidden from me

Reply:
Although the address is public, if there is a way to logically connect a public address to a real user, then the same user can become a target for a hacker that would like to discover the private key. 

This is the reason why other wallets hide the full address and display only part of it

Reply:
When you click on the "Copy" button, you'll still get the full address copied, right? So hiding a portion of the address won't make the address' owner less targetable

Reply:
The Use Case I am talking about is when a blogger takes a screen shot of his own page and displays it in an article , or when a youtuber records a video showing the sidebar when doing a video about how to use Polkadot, or someone taking a picture of the screen of his neighbor. So in these cases you can associate a real person (blogger, youtuber, or some one that is physically close) to a public address, which makes it more vulnerable. 
A person with bad intentions could know how much money has the account and then possibly decide to target it. 

The address would be copied by clicking on the button but still would not reveal it.

Reply:
yeah, I see your point